lib/hx509/cms.c

```diff
@@ -935,7 +935,7 @@ hx509_cms_verify_signed_ext(hx509_context context,
 	if (signer_info->signature.length == 0) {
 	    ret = HX509_CMS_MISSING_SIGNER_DATA;
 	    hx509_set_error_string(context, 0, ret,
-				   "SignerInfo %d in SignedData "
+				   "SignerInfo %zu in SignedData "
```

Comment:
Do old MSVC compilers and runtimes support this?

Reply:
I don’t have an old MSVC compiler to hand, sorry.

Reply:
But I see `%zu` is used in several places in Heimdal already.

Reply:
Ah, @jaltman tells me that roken supports `%zu`.